presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/measureresult/model/AlcoholType.kt

```diff
@@ -8,4 +8,18 @@ enum class AlcoholType(val title: String, val iconResId: Int) {
     KAOLIANGJU(title = "고량주", iconResId = R.drawable.ic_goryangju_s),
     WINE(title = "와인", iconResId = R.drawable.ic_wine_s),
     WHISKY(title = "위스키", iconResId = R.drawable.ic_whisky_s),
+    ;
+
+    companion object {
+        fun getType(name: String): AlcoholType {
+            return when (name) {
+                "소주" -> SOJU
+                "맥주" -> BEER
+                "와인" -> WINE
+                "위스키" -> WHISKY
+                "고량주" -> KAOLIANGJU
+                else -> throw IllegalArgumentException()
+            }
```

Comment:
```suggestion
        fun getType(name: String): AlcoholType = values().firstOrNull { it.name== name }
```

```suggestion
        fun getType(name: String): AlcoholType = values().associateBy(AlcoholType::title)[name]
```

요런식으로하면 enum이 추가되어도 getType 함수는 추가적으로 변경되지 않아도 돼

Reply:
반영했습니당 [d4ee728](https://github.com/mash-up-kr/SulSul-Android/pull/134/commits/d4ee728d55a0e4351e9611430170fcf86796e2df)